nautobot/docs/additional-features/caching.md

```diff
@@ -115,6 +115,48 @@ CACHES = {
 
 For more details on configuring django-redis with Redis Sentinel, please see the documentation for [Django Redis](https://github.com/jazzband/django-redis#use-the-sentinel-connection-factory).
 
+#### `django-redis` with TLS Sentinel Configuration
```

Comment:
I'd rather see this as a nested bullet under the previous section. I was a bit confused at first and had to double-take.

Like squeezing in the items that would go there between lines 112 and 113?